tests/test-extensions/test-oauth-extension/src/main/resources/META-INF/mule-artifact/mule-artifact.json

```diff
@@ -5,7 +5,7 @@
     "id": "java",
     "attributes": {
       "type": "org.mule.test.oauth.TestOAuthExtension",
-      "version": "4.6.0-SNAPSHOT"
+      "version": "4.0.0"
```

Comment:
?

Reply:
All the other mule-artifact.jsons seem to use a hardcoded version there too. For example: https://github.com/mulesoft/mule/blob/master/tests/test-extensions/test-localisation-properties-extension/src/main/resources/META-INF/mule-artifact/mule-artifact.json#L8